ccm_web/server/appHandler.ts

```diff
@@ -0,0 +1,103 @@
+import path from 'path'
+
+import express from 'express'
+import type { Express, Request, Response, Router } from 'express'
+import { IdToken, Provider } from 'ltijs'
+import Database from 'ltijs-sequelize'
+
+import { Config } from './config'
+
+interface EnvOptions {
+  isDev: boolean
+  staticPath: string
+}
+
+class AppHandler {
+  private readonly config: Config
+  private readonly apiRouter: Router
+  private readonly envOptions: EnvOptions
+
+  constructor (
+    config: Config,
+    envOptions: EnvOptions,
+    apiRouter: Router
+  ) {
+    this.config = config
+    this.envOptions = envOptions
+    this.apiRouter = apiRouter
+  }
+
+  // ltijs docs: https://cvmcosta.me/ltijs/#/
+  async setupLTI (): Promise<Express> {
+    const { db, lti } = this.config
+
+    const dbPlugin = new Database(
+      db.name,
+      db.user,
+      db.password,
+      { host: db.host, dialect: 'postgres', logging: false }
```

Comment:
Not sure what is this `logging: false` means, if this is for debugging more message then this should be coming from the `.env`. You can leave it for now since now a issue for merging this code

Reply:
This is a `sequelize` option. The `ltijs-sequelize` docs had `false` there. It likely prevents a bunch of SQL being echoed to the logs. It seems okay to me to have it be disabled (the default is `console.log`, looks like: https://sequelize.org/master/class/lib/sequelize.js~Sequelize.html), people can remove it or change the value to a function if they want to see the `ltijs` database operations.